src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -105,11 +105,16 @@ const isValidLabel = (type, value) => {
   return regex.test(value);
 };
 
+function resetError() {
```

Comment:
Is there a reason this is defined outside the component class?

Normally this would be defined like this:

```js
class CreatePipelineRun extends Component {
  constructor(props) {
    super(props);
    this.state = this.initialState(props);
  }

  resetError = () => {
    this.setState({ submitError: '' });
  }

  ...
}
```

Reply:
ah, nope, I've fixed that now for all of them